### ISSUE TITLE
can I know why use batch normalize after triplet loss and before softmax? why not use them with same feature map?

### ISSUE THREAD

**Issue by jorie-peng** [L1aoXingyu/reid_baseline#29]
the code is good and has a surprised result, but I am confused about loss, why use two loss with different feature map?

**Comment by L1aoXingyu** [comment]
you can check in this [repo](https://github.com/michuanhaohao/reid-strong-baseline) and corresponding paper.
